src/components/application_manager/src/commands/hmi/close_popup_request.cc

```diff
@@ -44,7 +44,11 @@ ClosePopupRequest::~ClosePopupRequest() {}
 
 void ClosePopupRequest::Run() {
   LOG4CXX_AUTO_TRACE(logger_);
-
+  if(!CheckAvailabilityHMIInterfaces(application_manager_,
```

Comment:
@pestOO Sorry I forget remove this checking from this place. I will remove this with next commit.